zanshinsdk/client.py

```diff
@@ -921,6 +921,14 @@ def check_organization_scan_target(self, organization_id: Union[UUID, str],
                              f"/organizations/{validate_uuid(organization_id)}/scantargets/"
                              f"{validate_uuid(scan_target_id)}/check").json()
 
+    def get_gworkspace_oauth_link(self) -> Dict:
+        """
+        Retrive a link to allow the user to authorize zanshin to read info from their gworkspace enviroment.
```

Comment:
Retrieve
environment

Reply:
you mean in the docstring?!

Reply:
Yeah. The spelling is wrong, that's all.

Reply:
oh ok, I'll correct that but just so you know I copied from the api (exactly the same) so it should be corrected there as well!

Reply:
Yikes. Do you know where it lives? Maybe you can correct that there too. Thanks.